geometry_msgs/Point position
geometry_msgs/Vector3 size
---
uint32[] object_ids